message_ix/report/__init__.py

```diff
@@ -141,6 +141,7 @@
         "message::costs",
         "message::emissions",
     ),
+    ("message::sankey", "concat", "out::pyam", "in::pyam"),
```

Comment:
I would get rid of this, and simply do the concatenation outside of reporter (e.g., in the jupyter notebook here)

Reply:
I kept this for now as I don't know what the Reporter actually does and it seems useful to just get the dataframe format we need from the Reporter. But please elaborate :)

Reply:
I have created a new file, sankey.py, within the report directory, which extracts Sankey diagram functionality from __init__.py. Fridolin (@glatterf42), could you please confirm if this is what we want? Additionally, I have updated the westeros_sankey.ipynb file to ensure compatibility with the new sankey.py file in the report directory. I have annotated these changes in the westeros_sankey.ipynb below. Also, in this case, the following line inside the __init__.py file must be either removed or commented out:
# ("message::sankey", "concat", "out::pyam", "in::pyam")

Reply:
Thanks, @daymontas1, these changes sound good to me. For the line that has become redundant, please remove it rather than commenting it out, this keeps the code clean.
Could you please push your changes to the branch so that we can take a look and see how the tests are doing?
If you don't have write access to @mabudz's fork, please let us know so that we can figure out a solution :)